### PR TITLE
[DOC-3407] Provide info about re-reading certificates on k8s-managed clusters

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-3407]

Reviewers: Let's try using only Github for this review and skipping Reviewable. Thanks!

### CODE REVIEW

**Review by pseudomuto — APPROVED** [review]
LGTM 👍

File: v21.2/create-security-certificates-custom-ca.md [hunk 1]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
-3. Add the `ui.crt` and `ui.key` to the [certificate directory](cockroach-cert.html#certificate-directory). `ui.key` must meet the [permission requirements check](cockroach-cert.html#key-file-permissions) on macOS, Linux, and other UNIX-like systems.
- .
-4. For nodes that are already running, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the cockroach process:
-   {% include_cached copy-clipboard.html %}
-   ~~~ shell
-   pkill -SIGHUP -x cockroach
-   ~~~
-   The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
+3. Add the `ui.crt` and `ui.key` to the [certificate directory](cockroach-cert.html#certificate-directory). `ui.key` must meet the [permission requirements check](cockroach-cert.html#key-file-permissions) on macOS, Linux, and other UNIX-like systems. If your cluster is deployed using containers, update the containers to include the new certificate and key.
+4. The cockroach process reads certificates only when the process starts.
```
Comment (taroface) [inline v21.2/create-security-certificates-custom-ca.md:38]: Nit: We usually style this:

```
`cockroach` process
```

File: v21.2/create-security-certificates-custom-ca.md [hunk 2]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
+
+   - In a manually-deployed cluster, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the cockroach process:
```
Comment (taroface) [inline v21.2/create-security-certificates-custom-ca.md:40]: Dito

File: v21.2/create-security-certificates-custom-ca.md [hunk 3]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
+      {% include_cached copy-clipboard.html %}
+      ~~~ shell
+      pkill -SIGHUP -x cockroach
+      ~~~
+      The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
```
Comment (taroface) [inline v21.2/create-security-certificates-custom-ca.md:45]: Ditto

File: v21.2/create-security-certificates-custom-ca.md (hunk 2 above)
Comment (taroface) [inline v21.2/create-security-certificates-custom-ca.md:40]: Maybe worth linking "manually-deployed cluster" to https://www.cockroachlabs.com/docs/stable/manual-deployment.html ?

File: v21.2/create-security-certificates-custom-ca.md [hunk 4]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
+   - In a cluster deployed using the Kubernetes Operator, it's not possible to send a `SIGHUP` signal to an individual cockroach process on each cluster node. Instead, perform a rolling restart of the cluster's pods.
```
Comment (taroface) [inline v21.2/create-security-certificates-custom-ca.md:46]: Perhaps link "deployed using the Kubernetes Operator" to https://www.cockroachlabs.com/docs/v22.1/deploy-cockroachdb-with-kubernetes

Also, per our [style](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md) let's avoid using contractions here (i.e., "in cases where a clear directive or prohibition is being given")

**Review by taroface — DISMISSED** [review]
A few nits on style. The v21.2 comments should apply to v22.1 as well.

Btw, any reason you didn't want to use Reviewable here? One of the benefits is that you get all comments at once, as opposed to being (I assume) individually notified for each note I left here.

File: v21.2/create-security-certificates-custom-ca.md (hunk 1 above)
Comment (mdlinville) [inline v21.2/create-security-certificates-custom-ca.md:38]:
```suggestion
4. The `cockroach` process reads certificates only when the process starts.
```

File: v21.2/create-security-certificates-custom-ca.md (hunk 2 above)
Comment (mdlinville) [reply v21.2/create-security-certificates-custom-ca.md:40]: Thanks for pointing these out. I was modeling it off the previous line 39.

Comment (mdlinville) [inline v21.2/create-security-certificates-custom-ca.md:40]:
```suggestion
   - In a manually-deployed cluster, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the `cockroach` process:
```

File: v21.2/create-security-certificates-custom-ca.md (hunk 4 above)
Comment (mdlinville) [inline v21.2/create-security-certificates-custom-ca.md:46]:
```suggestion
   - In a cluster deployed using the [Kubernetes Operator](/docs/stable/deploy-cockroachdb-with-kubernetes), there is no way to send a `SIGHUP` signal to the individual `cockroach` process on each cluster node. Instead, perform a rolling restart of the cluster's pods.
```

File: v22.1/create-security-certificates-custom-ca.md [hunk 5]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
+3. Add the `ui.crt` and `ui.key` to the [certificate directory](cockroach-cert.html#certificate-directory). `ui.key` must meet the [permission requirements check](cockroach-cert.html#key-file-permissions) on macOS, Linux, and other UNIX-like systems. If your cluster is deployed using containers, update the containers to include the new certificate and key.
+4. The cockroach process reads certificates only when the process starts.
+
+   - In a manually-deployed cluster, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the cockroach process:
+      {% include_cached copy-clipboard.html %}
+      ~~~ shell
+      pkill -SIGHUP -x cockroach
+      ~~~
+      The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
+   - In a cluster deployed using the Kubernetes Operator, it's not possible to send a `SIGHUP` signal to an individual cockroach process on each cluster node. Instead, perform a rolling restart of the cluster's pods.
```
Comment (mdlinville) [inline v22.1/create-security-certificates-custom-ca.md:46]:
```suggestion
   - In a cluster deployed using the [Kubernetes Operator](/docs/stable/deploy-cockroachdb-with-kubernetes), there is no way to send a `SIGHUP` signal to the individual `cockroach` process on each cluster node. Instead, perform a rolling restart of the cluster's pods.
```

File: v21.2/create-security-certificates-custom-ca.md [hunk 6]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
-   The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
+3. Add the `ui.crt` and `ui.key` to the [certificate directory](cockroach-cert.html#certificate-directory). `ui.key` must meet the [permission requirements check](cockroach-cert.html#key-file-permissions) on macOS, Linux, and other UNIX-like systems. If your cluster is deployed using containers, update the containers to include the new certificate and key.
+4. The `cockroach` process reads certificates only when the process starts.
+
+   - In a manually-deployed cluster, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the `cockroach` process:
+      {% include_cached copy-clipboard.html %}
+      ~~~ shell
+      pkill -SIGHUP -x cockroach
+      ~~~
+      The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
```
Comment (mdlinville) [inline v21.2/create-security-certificates-custom-ca.md:45]:
```suggestion
      The `SIGHUP` signal must be sent by the same user running the process (e.g., run with `sudo` if the `cockroach` process is running under user `root`).
```
Are we sure this is true? If it was started by an unprivileged user, could `root` send the signal, or would that re-launch the process under `root`?

File: v22.1/create-security-certificates-custom-ca.md [hunk 7]
```diff
@@ -30,18 +30,20 @@ Approach | Use case description
...
-   The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
+3. Add the `ui.crt` and `ui.key` to the [certificate directory](cockroach-cert.html#certificate-directory). `ui.key` must meet the [permission requirements check](cockroach-cert.html#key-file-permissions) on macOS, Linux, and other UNIX-like systems. If your cluster is deployed using containers, update the containers to include the new certificate and key.
+4. The cockroach process reads certificates only when the process starts.
+
+   - In a manually-deployed cluster, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the cockroach process:
+      {% include_cached copy-clipboard.html %}
+      ~~~ shell
+      pkill -SIGHUP -x cockroach
+      ~~~
+      The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
```
Comment (mdlinville) [inline v22.1/create-security-certificates-custom-ca.md:45]:
```suggestion
      The `SIGHUP` signal must be sent by the same user running the process (e.g., run with `sudo` if the `cockroach` process is running under user `root`).
```

File: v21.2/create-security-certificates-custom-ca.md (hunk 6 above)
Comment (mdlinville) [reply v21.2/create-security-certificates-custom-ca.md:45]: I filed https://cockroachlabs.atlassian.net/browse/DOC-4671 to explore the nuance of sending signals and privilege.

**Pull request merged**